src/todos.js

```diff
@@ -0,0 +1,30 @@
+export default class Todos {
+  constructor() {
+    this.list = localStorage.getItem('todos')
+      ? JSON.parse(localStorage.getItem('todos'))
+      : [];
+  }
+
+  addTodo(todo) {
+    this.list.push(todo);
+    localStorage.setItem('todos', JSON.stringify(this.list));
+  }
+
+  removeTodo(todoID) {
+    this.list = this.list.filter((todo) => todo.index !== todoID);
+    this.list.forEach((todo, index) => {
+      todo.index = index + 1;
+    });
+    localStorage.setItem('todos', JSON.stringify(this.list));
+  }
```

Comment:
- [ ] Kindly make sure When you remove tasks the last element also should be removed in the localstoreage. 

Uploading Screencast from 05-05-2022 12:33:51 AM.mp4…